### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/opt/carla-simulator/PythonAPI/examples/tensorflow_yolov3/carla/data\\classes\\coco.names'

### ISSUE THREAD

**Issue by amerfathullah** [umtclskn/Carla_Simulator_YOLOV3_Object_Detection#1]
HI,

I got this error when trying to run yolov3_object_detection.py . It said file not found, but the file exist in the directory.

Regards,
Amer

**Comment by RyanC1681** [comment]
same error as below. Did you fix it? Thanks
  File "C:\CARLA_0.9.11\WindowsNoEditor\PythonAPI\examples\tensorflow_yolov3\carla\utils.py", line 26, in read_class_names
    with open(class_file_name, 'r') as data:
FileNotFoundError: [Errno 2] No such file or directory: './data/classes/coco.names'

**Comment by amerfathullah** [comment]
no, I didn't. I just use other model. this repo has insufficient guide.

**Comment by sgyyds** [comment]
Do you have the file yolov3_coco.pb. I have the file of coco.names
